sympy/solvers/solveset.py

```diff
@@ -871,3 +872,162 @@ def solveset(f, symbol=None):
             return solveset_real(f, symbol)
         else:
             return solveset_complex(f, symbol)
+
+
+###############################################################################
+################################ LINSOLVE #####################################
+###############################################################################
+
+
+def eq_to_matrix(equations, *symbols):
+    """
+    Converts a given System of Equations into Matrix form.
+    Here `equations` must be a linear system of equations in
+    `symbols`. The order of symbols in input `symbols` will
+    determine the order of coefficients in the augmented
+    Matrix.
+
+    The Matrix form corresponds to the augmented matrix form.
+    For example:
+
+      x + 2.y + 3.z  = 1
+    3.x +   y +   z  = -6
+    2.x + 4.y + 9.z  = 2
+
+    This system would return:
+
+    [ 1  2  3   1 ]
+    [ 3  1  1  -6 ]
+    [ 2  4  9   2 ]
```

Comment:
Since, I was using the augmented Matrix in `linsolve` for converting it to reduced row echleon form `M.rref()` , I made it return the Augmented matrix.
Though, as you said there are more use cases for the former (A, b) I will make it return A & b.